src/System.Private.Uri/tests/FunctionalTests/IriTest.cs

```diff
@@ -184,7 +184,7 @@ public void Iri_IncompleteEscapedCharacter()
         [Fact]
         public void Iri_ReservedCharacters()
         {
-            string input = "/?#??#%[]";
+            string input = "?/#??#%[]";
```

Comment:
This changed because the original input would trigger recognition as a unix path?  Is there any concern about such things in the wild?